### ISSUE TITLE
Ability to have meeting notes in multiple areas

### ISSUE THREAD

**Issue by marksie1988** [dendronhq/dendron#2926]
## Please select if your request is either something new or an enhancement ##

- [x] Enhancement of an existing Feature.
- [ ] Request of a new feature.

## Please select the area your request applies to. (Multiple selections are Possible. You can leave blank if you're not sure.) ##

- [x] Workspace - VSCode workspace, vaults, Intellisense/autocomplete, Dendron settings
- [ ] Lookup - Dendron's Lookup Command
- [ ] Views - Dendron Preview, Tree View, Side Panels in the UI
- [x] Schema - Dendron Schemas
- [ ] Pod - Data import from / export to Dendron
- [ ] Publish - External Site Publish
- [ ] Markdown - Markdown features, such as syntax support and features

## Is your feature request related to a problem? Please describe ##

so basically the same as the current functionality but with the ability to specify the location the meeting note should be created without the need to type out the whole tree.

A clear and concise description of what the problem is. E.g. I'm always frustrated when [...]

## Describe the solution you'd like ##

I have a structure for projects: `p.myproject` I would like to be able to use the command pallete to add a new meeting note to different sections depending on the project the meeting relates to, this would then create a meeting note similar to `p.myproject.meet.2022.05.16.mymeeting`

**Comment by jonathanyeung** [comment]
Thank you @marksie1988 for the suggestion. We do plan on having enhancements later to make meeting notes more useful.
